WooCommerce/WooCommerceTests/POS/Card Present Payments/PointOfSaleCardPresentPaymentEventPresentationStyleTests.swift

```diff
@@ -93,7 +93,7 @@ final class PointOfSaleCardPresentPaymentEventPresentationStyleTests: XCTestCase
             return XCTFail("Expected payment success message not found")
         }
 
-        XCTAssertEqual(viewModel.message, "A card payment of $200.50 was successfully made")
+        XCTAssertEqual(viewModel.message, "A card payment of $200.50 was successfully made.")
```

Comment:
interesting, how do we guarantee the simulator is in English on CI?

Reply:
I'm not sure whether it's guaranteed, but it's relied upon in a bunch of places. Fastlane probably has a configuration file somewhere 😊 

I go back and forth on testing the string value, vs checking it matches the localized string in Constants. The latter is more resillient but a less useful and less readable test.